@xen-orchestra/web-core/lib/components/icon/VtsIcon.vue

```diff
@@ -0,0 +1,96 @@
+<!-- v2 -->
```

Comment:
Since the `Icon` component doesn't exist in the DS, maybe replace `v2` by `WiP`?